risc0/zkvm/platform/src/syscall.rs

```diff
@@ -29,6 +29,7 @@ pub mod nr {
     pub const SYS_LOG: u32 = 1;
     pub const SYS_IO: u32 = 2;
     pub const SYS_CYCLE_COUNT: u32 = 3;
+    pub const SYS_RAND: u32 = 4;
```

Comment:
Do we need a separate `SYS_RAND`? I was thinking we could build `getrandom` on top of `SYS_IO` with the `SENDRECV_CHANNEL_RANDOM` channel.

Reply:
Thanks for catching this! What you're suggesting is what I intended to do and this was left over from previous drafts. I'll remove it.

Reply:
Thanks! I made this adjustment